engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/mapper/RetriesDtoObjectMapperTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.rest.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.Date;
+import org.camunda.bpm.engine.rest.dto.history.HistoricProcessInstanceQueryDto;
+import org.camunda.bpm.engine.rest.dto.runtime.JobQueryDto;
+import org.camunda.bpm.engine.rest.dto.runtime.ProcessInstanceQueryDto;
+import org.camunda.bpm.engine.rest.dto.runtime.RetriesDto;
+import org.camunda.bpm.engine.rest.dto.runtime.SetJobRetriesByProcessDto;
+import org.camunda.bpm.engine.rest.dto.runtime.SetJobRetriesDto;
+import org.junit.Test;
+
+public class RetriesDtoObjectMapperTest {
+
+  @Test
+  public void shouldExcludeDueDateSetPropertyRetriesDto() throws JsonProcessingException {
+    // given
+    RetriesDto dto = new RetriesDto();
+    dto.setRetries(4);
+    dto.setDueDate(new Date(1675752840000L));
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    // when
+    String json = mapper.writeValueAsString(dto);
+
+    // then
+    assertThat(json).doesNotContain("dueDateSet");
+  }
```

Comment:
🔧 Can we add another test that deserializes a JSON string via Jackson, and the DTO is asserted? (the other way around).